src/content/pages/sponsor.mdx

```diff
@@ -0,0 +1,104 @@
+---
+title: Sponsor
+subtitle: Sponsoring EuroPython guarantees you highly targeted visibility and the opportunity to present yourself and your company to one of the largest and most diverse Python communities in Europe and beyond.
+---
+
+
+# Sponsor
+
+<font size="+2.5">Support the Largest European Python Conference!</font>
+
+Sponsoring EuroPython guarantees you highly targeted visibility and the opportunity to present yourself and your company to one of the largest and most diverse Python communities in Europe and beyond.
+
+Become a sponsor of EuroPython 2024. Join us this July in the **[Prague Congress Centre (PCC)](https://www.praguecc.cz/en/homepage)** and directly help promote the work of the great open-source community and further its development.
+
+<img src="/img/hall_rotated.png" width="3279" height="1785" />
+
+<ButtonWithTitle title="Ready to become a sponsor and join us in Prague?" text="Sign Up Now!" href="https://docs.google.com/forms/d/1Ut6b5KaYtj1hAivVxMdaSfeNCUCNTrTq98o5v9uNg6I/" />
+
+## 2023 Stats & Demographics
+
+EuroPython 2023 was a hybrid conference, both in Prague & Online, combining the good of both worlds.
+
+<div style={{textAlign: "center"}}>
+<ButtonLink href="https://drive.google.com/file/d/19kjdiSNDpK1COz2qYgJKMGUfa5Wzft5c/view?usp=sharing">2023 Stats & Demographics (PDF)</ButtonLink>
+</div>
+
+---
+ # Sponsor Tiers
+
+At EuroPython 2024, we offer seven main packages, aiming to keep EuroPython accessible to most organisations keen to help the community.
+
+<div style={{textAlign: "left", marginBottom: "30px"}}>
+<ButtonLink href="https://drive.google.com/file/d/1GZxkh04YFMJaWAswK_VXwdqmS7zAd2LC/view?usp=sharing">Sponsor Packages & Options Table (PDF)</ButtonLink>
+&nbsp;
+<ButtonLink href="https://forms.gle/ieK9LwdoRyDU4Z9a7">Sign up now!</ButtonLink>
+</div>
+
+<SponsorTiers />
+
+## Optional Extras!
```

Comment:
Can we plug in the description of the "Team5 + Patron", "Team10 + Patron", and "Team15 + Bronze" somehow? I don't know how exactly, but we could as ChatGPT 🤓